docs/quickstart.rst

```diff
@@ -23,28 +23,28 @@ To install the Python client library using pip:
 
 ::
 
-    pip install nexmo
+    pip install vonage
 
 Alternatively, you can clone the repository:
 
 ::
 
-    git clone git@github.com:Nexmo/nexmo-python.git
+    git clone git@github.com:Vonage/vonage-python.git
```

Comment:
This URL should be updated. It only works because Github will redirect it, but it should point to the correct URL (flagging each just so we don't miss them)